modules/main.nf

```diff
@@ -209,6 +209,23 @@ process index_bam{
     """
 }
 
+process generate_gene_counts4drop{
+
+	input:
+		tuple val(sample), path(counts)
+
+	output:
+		path('external_geneCounts.tsv'), emit: processed_gene_counts
```

Comment:
How would this work with a multisample run. Don't you need to either add sample? Like:
`tuple val(sample), path('external_geneCounts.tsv'), emit: processed_gene_counts`

Reply:
Nope! Here it's just one file we need from this process and its next one (sample annot process) parses the contents of the file for sample IDs. If in the case we don't have gene counts for a sample, then we'll take the tuple from output of STAR 😸